src/Sulu/Bundle/PageBundle/Search/EventSubscriber/BlameTimestampSubscriber.php

```diff
@@ -95,34 +95,27 @@ private function getFieldValue($document, $fieldName)
     /**
      * Map timestamps to the search document.
      */
-    private function mapTimestamp(Document $document, ?\DateTime $created = null, ?\DateTime $changed = null)
-    {
-        $document->addField(
-            $this->factory->createField('created', $created ? $created->format('c') : null, 'string')
-        );
-
-        $document->addField(
-            $this->factory->createField('changed', $changed ? $changed->format('c') : null, 'string')
-        );
+    private function mapTimestamp(
+        Document $document,
+        ?\DateTimeInterface $created = null,
+        ?\DateTimeInterface $changed = null,
+    ): void {
+        $document->addField($this->factory->createField('created', $created?->format('c'), 'string'));
+        $document->addField($this->factory->createField('changed', $changed?->format('c'), 'string'));
     }
 
     /**
      * Map the creator and changer to the document.
      */
-    private function mapCreatorAndChanger(Document $document, ?UserInterface $creator = null, ?UserInterface $changer = null)
-    {
-        $document->addField(
-            $this->factory->createField('changer', $changer ? $changer->getUserIdentifier() : null, 'string')
-        );
-        $document->addField(
-            $this->factory->createField('changer_id', $changer ? $changer->getId() : null, 'string')
-        );
-
-        $document->addField(
-            $this->factory->createField('creator', $creator ? $creator->getUserIdentifier() : null, 'string')
-        );
-        $document->addField(
-            $this->factory->createField('creator_id', $creator ? $creator->getId() : null, 'string')
-        );
+    private function mapCreatorAndChanger(
```

Comment:
here are no changes so this should be reverted?

Reply:
Code style changes better tackle in own PR else it can be get confusing with reviews.